src/SIM/SIM2D.C

```diff
@@ -81,8 +81,11 @@ bool SIM2D::parseGeometryTag (const TiXmlElement* elem)
     if (utl::getAttribute(elem,"lowerpatch",lowpatch))
       uppatch = myModel.size();
     utl::getAttribute(elem,"upperpatch",uppatch);
+    int check = myModel.size();
+    if (nGlPatches > 0)
+      check = nGlPatches;
```

Comment:
check er et noe dårlig variablenavn. Foreslår heller:

int nPatch = nGlPatches > 0 ? nGlPatches : myModel.size();

Gjelder også lengre nede i samme commit. Dessuten manglende this-> på getLocalPatchIndex kall.

Denne kommentaren gjler også for analoge endringer i SIM3D.C